tests/runtime/btf

```diff
@@ -91,3 +91,9 @@ EXPECT Attaching 1 probe...
 TIMEOUT 2
 REQUIRES_FEATURE kfunc
 REQUIRES lsmod | grep '^kvm'
+
+NAME kprobe_kernel_module_type_fwd
+PROG kprobe:kvm:x86_emulate_insn { $ctxt = (struct x86_emulate_ctxt *) arg0; printf("%d\n", $ctxt->mode); exit(); }
+EXPECT Attaching 1 probe...
+TIMEOUT 2
+REQUIRES lsmod | grep '^kvm'
```

Comment:
Is it possible to write a BEFORE program to load the kvm module if it's not already loaded?

Reply:
Good idea. I'll do that in another PR, since I'd rather not sneak it into this one; there may be a good reason that it's not already done.